src/app/layout.tsx

```diff
@@ -13,37 +14,35 @@ export default function RootLayout({
 }>) {
   return (
     <html lang="en">
-         <head>
+      <head>
         <meta property="og:title" content="Farcaster Explorer" />
-        <meta property="og:description" content="Explore event propogation on farcaster." />
+        <meta property="og:description" content="Explore event propagation on farcaster." />
         <meta property="og:image" content="/neynar.png" />
         <meta property="og:url" content="https://www.neynar.com" />
         <meta property="og:type" content="website" />
         <meta name="twitter:card" content="summary_large_image" />
         <meta name="twitter:title" content="Farcaster Explorer" />
-        <meta name="twitter:description" content="Explore event propogation on farcaster." />
+        <meta name="twitter:description" content="Explore event propagation on farcaster." />
         <meta name="twitter:image" content="/neynar.png" />
       </head>
-      <body className="relative h-screen flex flex-col items-center justify-center min-h-screen">
+      <body className="relative h-screen flex flex-col items-center min-h-screen">
         <Providers>
-          <div className="absolute top-0 left-0 p-2">
-            <Link href="https://www.neynar.com" target="_blank">
-              <img className="w-16 md:w-32" src={"/neynar.png"} />
-            </Link>
-          </div>
-          <div className="absolute top-5 left-1/2 transform -translate-x-1/2 p-2">
-            <Link href="/" className="text-black font-bold">
-              <p className="text-md md:text-">Farcaster Explorer</p>
-            </Link>
-          </div>
-          <div className="w-full absolute top-10 justify-center mt-12 max-h-32 max-w-sm md:max-w-md">
-            <Search />
-      </div>
-          <div className="w-full px-4 md:px-8 mt-24 md:mt-0 flex flex-col ">
+          <div className="w-1/2 mt-5 px-4 md:px-8 flex flex-col items-center">
+            <div className="m-auto relative">
+              <Link href="https://www.neynar.com" target="_blank">
+                  <img className="w-16 md:w-32" src="/neynar.png" />
+              </Link>
+            </div>
+            <Card className="w-[100%] p-3">
+              <Link href="/" className="text-black font-bold">
+                <p className="mb-4 text-lg md:text-">Farcaster Explorer</p>
+              </Link>
+              <Search />
+            </Card>
```

Comment:
can you change this to what it was before?

on the top left, the neynar logo should show, in the center is farcaster explorer